doc/classes/Button.xml

```diff
@@ -57,9 +57,18 @@
 		<member name="icon" type="Texture2D" setter="set_button_icon" getter="get_button_icon">
 			Button's icon, if text is present the icon will be placed before the text.
 		</member>
+		<member name="language" type="String" setter="set_language" getter="get_language" default="&quot;&quot;">
+			Language code used for line-breaking and text shaping algorithms, if left empty current locale is used instead.
+		</member>
+		<member name="ot_features" type="String" setter="set_ot_features" getter="get_ot_features" default="&quot;&quot;">
+			Comma separated list of OpenType feature tags. More info: [url=https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags]OpenType feature tags[/url].
+		</member>
 		<member name="text" type="String" setter="set_text" getter="get_text" default="&quot;&quot;">
 			The button's text that will be displayed inside the button's area.
 		</member>
+		<member name="text_direction" type="int" setter="set_text_direction" getter="get_text_direction" enum="Control.TextDirection" default="-1">
```

Comment:
Shouldn't this be deduced from the font resource used? I believe it may be annoying to have to set manually no?

Reply:
It's deduced from locale normally (default value), this is for override (in case you need text in the other language in some controls) usually there's no need to change it. It can't be deduced from the font since most fonts for RTL languages have Latin characters as well (only big fontsets with the huge coverage, like Noto have separate files for each language).

Reply:
Then I'd maybe rename it to text_direction_override. It would make it more obvious that this otherwise deduced automatically.